src/Day01.kt

```diff
@@ -2,23 +2,25 @@
 fun main() {
     fun part1(input: List<String>): Int {
         var checkSum = 0
-        for (values in input) {
-            val num = values.split(' ')
-            checkSum += num.maxOf { it.toInt() } - num.minOf { it.toInt() }
+        for (line in input) {
```

Comment:
`values` isn't a terrible name, but since you're then taking what is stored in `values` and processing it further (to get `num`, originally), I think it's better to make this variable name something more clearly associated with it's purpose. What you're really interested in this `for` loop is getting a each line of input individually.